setup.cfg

```diff
@@ -14,6 +14,7 @@ zip_safe = False
 python_requires = >=3.6
 packages = find:
 install_requires =
+    astropy!=4.0.1
```

Comment:
```suggestion
    astropy
```

Reply:
Let see if it works this time now that a new release of `astropy` is out...